src/resolve/resolved-dns-stub.c

```diff
@@ -949,6 +950,21 @@ static void dns_stub_process_query(Manager *m, DnsStubListenerExtra *l, DnsStrea
                 return;
         }
 
+        if (m->refuse_record_types) {
+                STRV_FOREACH(i, strv_split(m->refuse_record_types, ",")) {
+                        int refuse_record_type;
+                        // delete_chars is just in case the user accidentally use qoutes like "AAAA" instead of AAAA in configuration
+                        char* record_type_string = delete_chars(*i,"\"");
```

Comment:
missing space after `*i,`

Reply:
using `extract_first_word` now. this block is removed.